Yosemite/YosemiteTests/Stores/ProductStoreTests.swift

```diff
@@ -969,9 +976,11 @@ final class ProductStoreTests: XCTestCase {
         let productStore = ProductStore(dispatcher: dispatcher, storageManager: storageManager, network: network)
 
         let product = sampleProduct()
-        let action = ProductAction.updateProduct(product: product) { (product, error) in
-            XCTAssertNotNil(error)
-            XCTAssertNil(product)
+        let action = ProductAction.updateProduct(product: product) { result in
```

Comment:
same here 👍

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/2533/commits/9f6c5b1d6666bbd75ba2fa8564cbd4bcd2bd93a7 😄